command/var_list_test.go

```diff
@@ -13,13 +13,15 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+// cSpell:ignore Pather
```

Comment:
Do we run this in CI?

Reply:
Not _yet_. It was some unintended bleedover from some code quality work I was doing in another branch.